### ISSUE TITLE
Language with empty translations

### ISSUE THREAD

**Issue by wojciechkryg** [hyperdevs-team/poeditor-android-gradle-plugin#23]
**SW details (please complete the following information):**
 - IDE version: Android Studio 4.1.1
 - Plug-in Version 1.4.2

**Summary and background of the bug**
Languages with empty translations returns empty updated data string from POEditor API which causes an error.

JSON fetched from POEditor:
```
{
   "response":{
      "status":"success",
      "code":"200",
      "message":"OK"
   },
   "result":{
      "languages":[
         {
            "name":"English",
            "code":"en",
            "translations":2,
            "percentage":100,
            "updated":"2021-02-22T07:32:39+0000"
         },
         {
            "name":"German",
            "code":"de",
            "translations":0,
            "percentage":0,
            "updated":""
         }
      ]
   }
}
```

**Steps to reproduce**
1. Set up empty language without translations in POEditor
2. Try to download translations

**Stacktrace**
```
Caused by: java.text.ParseException: Unparseable date: ""
        at com.bq.poeditor.gradle.utils.DateJsonAdapter.fromJson(DateJsonAdapter.kt:39)
        at com.bq.poeditor.gradle.utils.DateJsonAdapter.fromJson(DateJsonAdapter.kt:29)
```

**Expected behavior**
Download empty translations without any problem.

**Current behavior**
Error occurs `java.text.ParseException: Unparseable date: ""`.

**Comment by adriangl** [comment]
Hi! I apologize for the late reply, we'll review the issue and the PR ASAP!

**Comment by adriangl** [comment]
Closed in #24 and #28
